Presentation.Web/app/models/generic/BusinessRoleDTO.ts

```diff
@@ -0,0 +1,8 @@
+﻿module Kitos.Models.Generic.Roles {
+
+    export interface BusinessRoleDTO extends Generic.NamedEntity.NamedEntityDTO {
```

Comment:
Hvad er en BusinessRole? Og hvis den bruges mere generelt ville det så ikke give mening at det bare var en Role så vi har den ene type i stedet for at extende den når der så ikke tilføjes nyt i extension?

Reply:
en business role er et generelt begreb vi allerede har generaliseret i backend. Modsat en organisationsrolle er den forretnings/domænebestemt :). En organisationsrolle er noget helt andet og kan ikke entydigt give skriveadgang eller udfases. NamedEntityDTO er base class i de her hierarkier så vi ikke opfinder alting på ny i vores DTO snit